test/helpers/ECRecoveryMock.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.11;
+
+
+import '../../contracts/ECRecovery.sol';
+
+
+contract ECRecoveryMock {
+
+  bool public result;
+  address public signer;
+
+  function safeRecover(bytes32 hash, uint8 v, bytes32 r, bytes32 s) {
+    (result, signer) = ECRecovery.safeRecover(hash, v, r, s);
+  }
+
+  function recover(bytes32 hash, bytes sig) {
+    signer = ECRecovery.recover(hash, sig);
+  }
+
+}
```

Comment:
Try doing it without a mock, it should work. See: https://github.com/OpenZeppelin/zeppelin-solidity/issues/259